ga4gh/static/swagger.json

```diff
@@ -0,0 +1,172 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "GA4GH Reference Server",
+    "version": "0.2.0",
+    "title": "GA4GH Reference Server",
+    "contact": {
+      "email": "someone@ucsc.edu"
+    },
+    "license": {
+      "name": "Apache 2.0",
+      "url": "http://www.apache.org/licenses/LICENSE-2.0.html"
+    }
+  },
+  "host": "localhost:8000",
+  "tags": [
+    {
+      "name": "variants",
+      "description": "Variant endpoints",
+      "externalDocs": {
+        "description": "Find out more",
+        "url": "documentation site"
+      }
+    }
+  ],
+  "schemes": [
+    "http"
+  ],
+  "paths": {
+    "/variants/{id}": {
+      "get": {
+        "tags": [
+          "variants"
+        ],
+        "description": "Returns variant for a given ID",
+        "summary": "Find variants by ID",
+        "operationId": "getVariant",
+        "produces": [
```

Comment:
We need an approach to building the documentation from the schema source rather than need to maintain it in to places?